### DISCUSSION TITLE
Conditional file inclusion

### DISCUSSION THREAD

**Discussion by randrej** [copier-org/copier#455] [Q&A] [answered]
Is there a way to conditionally include a file depending on the value of a variable?

**Comment by yajo** [comment]
What do you mean with "include"? Jinja include? Render the whole file conditionally? Something else?

**Comment by randrej** [comment]
Render the whole file conditionally.

If a variable `abc = True`, the file `test.txt` is rendered into the output directory. If it's false, it's not rendered into the output directory.

**Comment by yajo** [comment, marked as answer]
Just template the file name. Call it something like: `{% if abc %}myfile.txt{% endif %}`

If the file/dir name become empty after applying templating, it will be skipped.
